vendor/vendor.json

```diff
@@ -661,6 +661,12 @@
 			"versionExact": "v1.10.44"
 		},
 		{
+			"checksumSHA1": "TsLkr/jtNxQY3RSJjbCZXEaopG0=",
+			"path": "github.com/aws/aws-sdk-go/service/servicecatalog",
+			"revision": "f43761405cff341da21289c4f14800c41ccdc10b",
+			"revisionTime": "2017-08-25T16:44:15Z"
```

Comment:
I cannot see the version here, like `v1.10.44`
Can you please provide the govendor command you used?

Thanks! 😄